modules/assim.sequential/R/load_data_paleon_sda.R

```diff
@@ -0,0 +1,148 @@
+##' @title load_data_paleon_sda
+##' @name  load_data_paleon_sda
+##' @author Ann Raiho \email{araiho@nd.edu}
+##' 
+##' @param settings    PEcAn SDA settings object
+##' 
+##' @description Load data function for paleon SDA data products
+##' 
+##' @return obs.mean and obs.cov for sda.enkf
+##' @export
+##' 
+load_data_paleon_sda <- function(settings){
+  
+  #### Types of Data needed
+  ## STEPPS Fcomp
+  ## ReFAB biomass
+  ## HF Tree Rings (CHECK)
+  ## Tree Rings from tree ring module
+  ## PLS and FIA Biomass Snapshot
+  ## Wish list: eddy covariance
+  
+  library(plyr) #need to load to use .fnc below
+  
+  d <- settings$database$bety[c("dbname", "password", "host", "user")]
+  bety <- src_postgres(host = d$host, user = d$user, password = d$password, dbname = d$dbname)
+  settings$host$name <- "localhost"
+  
+  site <- PEcAn.DB::query.site(settings$run$site$id, bety$con)
+  format_id <- settings$state.data.assimilation$data$format_id
+  
+  input.list <- list()
+  input.id <- list()
+  obvs <- list()
+  
+  var.names <- unlist(sapply(settings$state.data.assimilation$state.variable, 
+                             function(x) {
+                               x$variable.name
+                             }, 
+                             USE.NAMES = FALSE), 
+                      use.names = FALSE)
+  
+  start_date <- settings$state.data.assimilation$start.date
+  end_date   <- settings$state.data.assimilation$end.date
+  
+  obs.mean <- list()
+  obs.cov <- list()
+  
+  start.time <- format(lubridate::ymd(start_date),settings$state.data.assimilation$forecast.time.step)
+  end.time <- format(lubridate::ymd(end_date),settings$state.data.assimilation$forecast.time.step)
+  obs.times <- start.time:end.time
```

Comment:
So how would that work if the timestep was, for example, 8 days like we did in the MODIS analysis? Would you write ```8 %d```? or ```8*%d```? Would lubridate or difftime or anything else be able to parse that automatically?